packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -132,6 +133,50 @@ describe('Dialog', () => {
           expect(message).not.toBeUndefined();
           expect(message.args).toStrictEqual(['exampleMessage']);
         });
+
+        it(`FRAMED: Should successfully call the callback when dialog.closed: ${context}`, async () => {
+          await framedMock.initializeWithContext(FrameContexts.content);
+          const urlDialogInfo: UrlDialogInfo = {
+            url: 'someUrl',
+            size: { height: DialogDimension.Large, width: DialogDimension.Large },
+            title: 'someTitle',
+            fallbackUrl: 'someFallbackUrl',
+          };
+          const submitString = 'succesfullySubmit';
+
```

Comment:
Can you add assertions to make sure `try catch` is executed once?

Reply:
I don't expect it to go to the catch block. So, it won't hit any expect statement. 
That is the reason, I think assertions would not work on these tests. 
Feel free to show me an example of how it can be done (if I am missing something).

Reply:
@erinha @shrshindeMSFT